### PR TITLE
use `::class` constant for model generator

### PR DESCRIPTION
**Pull request by Faryshta**
| Q | A |
| --- | --- |
| Is bugfix? | no |
| New feature? | no |
| Breaks BC? | no |
| Tests pass? | yes |

### CODE REVIEW

File: generators/model/default/model.php [hunk 1]
```diff
@@ -93,7 +93,7 @@ public function get<?= $name ?>()
      */
     public static function find()
     {
-        return new <?= $queryClassFullName ?>(get_called_class());
+        return new <?= $queryClassFullName ?>(static::class);
```
Comment (cebe) [inline generators/model/default/model.php]: are you sure this works? `::class` is a compile time feature while `static` should work dynamically dependent on the call.

Comment (Faryshta) [inline generators/model/default/model.php]: https://3v4l.org/0Yb1T

Comment (cebe) [inline generators/model/default/model.php]: okay. still I find this a bit cryptic. get_called_class() is more obvious imo.

Comment (rob006) [inline generators/model/default/model.php]: > but get get_called_class() is buggy. for example https://3v4l.org/LUafm

https://3v4l.org/Pdko5

**Comment by klimov-paul** [comment]
Resolved by commit aa34d2174b3d1cc89fd9e80edbbc99460babea17

**Pull request closed without merging**